pennylane/drawer/tape_text.py

```diff
@@ -127,16 +127,18 @@ def tape_text(
 
     .. code-block:: python
 
-        with qml.tape.QuantumTape() as tape:
-            qml.QFT(wires=(0, 1, 2))
-            qml.RX(1.234, wires=0)
-            qml.RY(1.234, wires=1)
-            qml.RZ(1.234, wires=2)
+        ops = [qml.QFT(wires=(0, 1, 2)),
+            qml.RX(1.234, wires=0),
```

Comment:
I think the `QFT` should be on the next line.
```suggestion
        ops = [
            qml.QFT(wires=(0, 1, 2)),
            qml.RX(1.234, wires=0),
```